CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Tabstops not being reset with `reset`
 - Selection not cleared when switching between main and alt grid
 - Fallback to `LC_CTYPE=UTF-8` on macOS without valid system locale
+- Numbers are now allowed as mouse binding value
```

Comment:
If this is a new feature, it should be part of `Added`. If it's a fix for an old issue it should be properly documented.

I think I'd move it to changed section and go into more detail that mouse bindings are accepted again, but only as numbers, not as strings.